pennylane/qnodes/passthru.py

```diff
@@ -102,44 +116,13 @@ def _construct(self, args, kwargs):
         del self.queue
         del self.obs_queue
 
+        # no output conversion
+        self.output_conversion = lambda x: x
```

Comment:
Since this is independent of the circuit construction, better to go in `__init__`?

Reply:
It has to override the `output_conversion` set by `_check_circuit`